skimage/transform/_warps.py

```diff
@@ -66,8 +66,9 @@ def resize(image, output_shape, order=None, mode='reflect', cval=0, clip=True,
     anti_aliasing : bool, optional
         Whether to apply a Gaussian filter to smooth the image prior
         to down-scaling. It is crucial to filter when down-sampling
-        the image to avoid aliasing artifacts. If input image data
-        type is bool, no anti-aliasing is applied.
+        the image to avoid aliasing artifacts. If input data type is not bool,
+        anti_aliasing is applied by default when down-sampling. If input image
+        data type is bool, no anti-aliasing is applied.
```

Comment:
```suggestion
        the image to avoid aliasing artifacts. Default is True if down-scaling and
        image data type is not bool, and False otherwise.
```

This behavior can also be explicit in the code line 109:
```python
if anti_aliasing is None:
    anti_aliasing = (not image.dtype == bool
                     and np.all(out_shape < in_shape))
```